assets/src/dashboard/utils/usePagePreviewSize.js

```diff
@@ -99,17 +100,24 @@ const sizeFromWidth = (
   };
 };
 
+const getContainerWidth = (windowWidth) => {
+  return windowWidth >= MIN_DASHBOARD_WIDTH ? windowWidth : MIN_DASHBOARD_WIDTH;
+};
+
```

Comment:
Because we have a min width in the dashboard now we actually don't want to know what size the screen is when it's smaller than `MIN_DASHBOARD_WIDTH`, so we're just checking here now to see if the window's smaller than the width, if it is we're just going to use the min dashboard width as our base for calculating so that the grid items stay even with the rest of the page.